### ISSUE TITLE
Cells require second tap to register (Springboard)

### ISSUE THREAD

**Issue by iamcam** [AlanQuatermain/AQGridView#65]
Using the springboard as a starting point to replace a UITableView, selecting a cell the first time works (takes you to the next controller), but when you return back to the springboard and tap on the same cell, the first touch does not register in `gridView:didSelectItemAtIndex:`. A second tap registers. If a different cell is tapped, it works fine. Also taking the push to another view controller out of the equation still results in the same behavior:
first tap: activate
second tap: nothing
third: activate
fourth: nothing
etc...

So basically, it seems a cell becomes deactivated and reactivated with a second tap.

I am using a large portion of the Springboard sample code for layout and gesture recognition.

**Comment by iamcam** [comment]
After a brief discussion with Alan, it turns out this is the expected behavior because it allows you to select/deselect a cell. To get it to perform like a UITableView (as far as I'm concerned), you need to use `[gridView deselectItemAtIndex: index animated: NO];` or similar in your view controller's `gridView:didSelectItemAtIndex:` method.
